### ISSUE TITLE
Uncaught TypeError: Cannot read property 'id' of undefined

### ISSUE THREAD

**Issue by VladimirHumeniuk** [steelbrain/linter-ui-default#361]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0-beta0 x64
**Electron**: 1.6.9
**OS**: Linux Mint
**Thrown From**: [linter-ui-default](https://github.com/steelbrain/linter-ui-default) package 1.6.1

### Stack Trace

Uncaught TypeError: Cannot read property 'id' of undefined

```
At /usr/share/atom-beta/resources/app/src/text-editor-component.js:2132

TypeError: Cannot read property 'id' of undefined
    at horizontalPositionsToMeasure.forEach (/usr/share/atom-beta/resources/app/src/text-editor-component.js:2132:73)
    at Map.forEach (native)
    at TextEditorComponent.measureHorizontalPositions (/usr/share/atom-beta/resources/app/src/text-editor-component.js:2128:45)
    at TextEditorComponent.measureContentDuringUpdateSync (/usr/share/atom-beta/resources/app/src/text-editor-component.js:373:16)
    at TextEditorComponent.updateSync (/usr/share/atom-beta/resources/app/src/text-editor-component.js:270:18)
    at TextEditorComponent.pixelPositionForScreenPosition (/usr/share/atom-beta/resources/app/src/text-editor-component.js:194:18)
    at HTMLElement.pixelPositionForScreenPosition (/usr/share/atom-beta/resources/app/src/text-editor-element.js:243:32)
    at getBufferPositionFromMouseEvent (/packages/linter-ui-default/lib/editor/helpers.js:12:47)
    at /packages/linter-ui-default/lib/editor/index.js:163:29)
    at later (/packages/linter/node_modules/sb-debounce/index.js:9:14)
```

### Commands

```
     -1:10.9.0 intentions:highlight (input.hidden-input)
     -1:10.7.0 core:select-all (input.hidden-input)
     -1:09.4.0 intentions:highlight (input.hidden-input)
     -1:09.3.0 core:select-all (input.hidden-input)
     -1:08.9.0 core:paste (input.hidden-input)
     -1:01.3.0 intentions:highlight (input.hidden-input)
     -1:01.2.0 core:undo (input.hidden-input)
     -1:00.4.0 intentions:highlight (input.hidden-input)
     -1:00.4.0 core:select-all (input.hidden-input)
     -0:58.3.0 intentions:highlight (input.hidden-input)
     -0:58.2.0 core:select-all (input.hidden-input)
     -0:56.1.0 intentions:highlight (input.hidden-input)
     -0:56 core:paste (input.hidden-input)
     -0:53.7.0 intentions:highlight (input.hidden-input)
     -0:53.5.0 core:save (input.hidden-input)
     -0:21 git-time-machine:toggle (atom-workspace.workspace.scrollbars-visible-always.seti-compact.seti-icons.seti-roboto.seti-animate.theme-atom-monokai.theme-seti-ui.find-visible)
```

### Non-Core Packages

```
aligner 1.2.2 
aligner-javascript 1.1.0 
apex-ui 0.2.1 
atom-beautify 0.30.1 
atom-clock 0.1.10 
atom-material-syntax 1.0.6 
atom-material-syntax-dark 1.0.0 
atom-material-ui 2.0.4 
atom-monokai 0.10.9 
atom-typescript 11.0.3 
autoclose-html 0.23.0 
autocomplete-paths 1.0.5 
busy-signal 1.4.3 
dimmed-monokai 0.1.0 
emmet 2.4.3 
find-trailing-whitespace 0.1.0 
git-time-machine 1.5.9 
gruvbox-plus-syntax 1.3.1 
Hydrogen 1.17.1 
intentions 1.1.2 
language-babel 2.66.0 
language-postcss 1.3.1 
language-pug 0.0.21 
linter 2.2.0 
linter-eslint 8.2.1 
linter-ui-default 1.6.1 
material-monokai-syntax 1.0.1 
material-ui 1.0.10 
monokai-seti 0.9.0 
pigments 0.39.1 
platformio-ide-terminal 2.5.1 
pristine-ui 1.0.15 
pure-syntax 0.2.1 
seti-classic 0.10.3 
seti-syntax 1.1.3 
seti-ui 1.8.0 
seti-ui-red 0.13.0 
tabs-to-spaces 1.0.3 
toggle-quotes 1.0.1 
```

**Comment by steelbrain** [comment]
You are on Atom beta, revert to stable if you don't like seeing this

**Comment by Babazon** [comment]
<img width="1511" alt="screen shot 2018-01-01 at 14 37 16" src="https://user-images.githubusercontent.com/9430138/34467690-565e521a-ef01-11e7-9dfd-013422fcccd7.png">

I am on stable build and get the same error and get directed to this issue. 
This has begun after I installed atom-linter-json-lint package and while editing the JSON file. 
( https://github.com/josa42/atom-linter-json-lint )

**Comment by Arcanemagus** [comment]
For reference this particular issue should have been fixed by https://github.com/atom/atom/pull/15051, released in Atom v1.20.0.
